dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/EventPersister.java

```diff
@@ -105,6 +106,15 @@ protected boolean isNew(TrackerPreheat preheat, String uid) {
 
   @Override
   protected TrackerSideEffectDataBundle handleSideEffects(TrackerBundle bundle, Event event) {
+
+    SideEffectTrigger trigger = SideEffectTrigger.NONE;
+
+    if (isNew(bundle.getPreheat(), event.getUid())) {
+      trigger = SideEffectTrigger.EVENT_UPDATE;
```

Comment:
Why is this called `EVENT_UPDATE`?
This is triggered when a new event is created